CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ The minor version will be incremented upon a breaking change and the patch versi
 
 ### Breaking
 
+## [0.30.1] - 2024-04-17
+
+### Fixes
+
+- syn: Eliminate variable allocations that build up stack space for token extension code generation ([#2913](https://github.com/coral-xyz/anchor/pull/2913)).
+
```

Comment:
Could you add this change under the [Unreleased](https://github.com/coral-xyz/anchor/blob/8e8df76eda965ca6815dbdcb27f96c3dea018b3f/CHANGELOG.md#unreleased) section, and remove the new version section?